### PR TITLE
Add profile picture of complete faculty staff

### LINKED ISSUE #10: Add profile picture of complete faculty staff
Add a folder containing profile picture of complete faculty staff

The final folder should look like [this](https://github.com/GOVINDDIXIT/IIITA-Student-App/tree/master/faculty/photos)

### PR DESCRIPTION
**Pull request by Retarded-Ape**
fix #10 
Since, 3 days are over and the previous person who was assigned this task was not successful in fixing this issue, here is my PR fixing this issue.

### CODE REVIEW

**Comment by GOVINDDIXIT** [comment]
@Retarded-Ape There is a pending PR with a deadline of today. You will be assigned after the 72 hours deadline is over.

**Comment by Retarded-Ape** [comment]
72 hours are are over sir, Please check my PR

File: blogs/GETSETFOSS.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (GOVINDDIXIT) [inline blogs/GETSETFOSS.md]: remove this

Comment (GOVINDDIXIT) [reply blogs/GETSETFOSS.md]: Also, comment on the issue first, It is now open for everyone

**Pull request closed without merging**